### DISCUSSION TITLE
Resolve symbolic links for workflows

### DISCUSSION THREAD

**Discussion by danra** [community/community#109744] [Actions]
It would be useful if GitHub would resolve symbolic links when parsing workflows in `.github/workflows`. Besides generally allowing more flexibility in where workflows are actually placed, this would specifically allow placing common workflows in a common repo which could than be brought into client repos as a submodule or otherwise. Note this would allow reusing workflow dispatch conditions, not just actions; this isn't something that is resolved just by using `uses:`.

Curently, placing a symbolic link in `.github/workflows` just fails GitHub's CI immediately.

See also https://stackoverflow.com/questions/70682565/can-i-symlink-github-actions-to-directories-inside-repo

**Comment by silverwind** [comment]
Same issue. I would like to symlink the whole `.github` folder, but as soon as I do, Actions will no longer run.

Example repo: https://github.com/silverwind/symlink-github

As per https://github.com/actions/runner/issues/3195, the bug is apparently not in the runner.
